torch/ao/quantization/backend_config/native.py

```diff
@@ -260,6 +261,55 @@ def _get_linear_configs(dtype_configs):
         "root_module": torch.nn.Linear,
         "reference_quantized_module_for_root": nnqr.Linear,
     })
+
+    # (4) Linear + leaky_relu
```

Comment:
wondering if we can dedup the code here as well, there must be some duplications with linear - relu pattern. I'm moving out this part of the code in https://github.com/pytorch/pytorch/pull/78407 maybe you can wait a little bit and rebase on that

Reply:
OK